src/app/bossnomics/section-3.tsx

```diff
@@ -1,56 +1,66 @@
 import { Typography, Stack } from '@mui/joy';
-import { GrantsTable } from '@/shared/components/grants-table';
+import { HeroSectionWithOverflow } from '@/shared/components/hero-section-with-overflow';
+import { TableGrants } from '@/shared/components/table-grants';
+
+const investorData = [
```

Comment:
@PupoSDC this shouldn't be here.

there should only be 1 row saying: 'Website & Brand Grant | Vesting | 2%'